vendor/vendor.json

```diff
@@ -146,10 +138,20 @@
 			"revisionTime": "2019-06-13T14:39:42Z"
 		},
 		{
-			"checksumSHA1": "z6OakgyT4iR/xAKnnko9jXdJPlI=",
+			"checksumSHA1": "3HWurU2V4/n3BjF7UpJu132zGrk=",
+			"origin": "github.com/mendersoftware/cli",
 			"path": "github.com/urfave/cli",
-			"revision": "e6cf83ec39f6e1158ced1927d4ed14578fda8edb",
-			"revisionTime": "2019-08-04T02:37:52Z"
+			"revision": "2eb686fe967b14b3e204f0eff66427e1fdafcf6d",
+			"revisionTime": "2020-02-18T09:03:34Z",
+			"tree": true,
+			"version": "v2.1.1-minimal",
+			"versionExact": "v2.1.1-minimal"
+		},
+		{
+			"path": "github.com/urfave/cli/v2",
```

Comment:
Can we remove this entry? The only file that depends on this path is ignored.

Reply:
Sure, `govendor` added this automatically.